lib/workers/repository/update/branch/index.ts

```diff
@@ -852,7 +853,11 @@ export async function processBranch(
       } else if (config.automerge) {
         logger.debug('PR is configured for automerge');
         // skip automerge if there is a new commit since status checks aren't done yet
-        if (!commitSha || config.ignoreTests) {
+        if (
+          !commitSha ||
+          // hard to reach
+          /*istanbul ignore next */ config.ignoreTests
```

Comment:
```suggestion
          /* istanbul ignore next: hard to reach */ config.ignoreTests
```
use single comment style 😉
probably needs prettier.

maybe reorder conditions to get coverage? 🙃